XLS-0033d-multi-purpose-tokens/README.md

```diff
@@ -834,6 +834,10 @@ Some benefits of this design approach include that is (1) reduces the footprint
 #### A.1.14. Why doesn’t an `MPTokenIssuanceID` simply hash an issuer address and currency code?
 Primarily because we did not want MPT meta-data (e.g., currency code, asset precision, common name, currency symbol, etc.) to be part of an MPTs unique identifier. This mirrors the design of other tokenization primitives on other blockchain networks that have gained massive adoption, offering strong “prior art” (e.g., ERC-20 and ERC-721 tokens). For a more detailed discussion, see [here](https://github.com/XRPLF/XRPL-Standards/discussions/128).
 
+#### A.1.15. Why does `MPTokenIssuance` include an `AssetCode` instead of letting this exist solely in `MPTokenMetadata`?
```

Comment:
did you mean AssetScale instead of AssetCode here?

Reply:
Doh! Yes, nice catch. Fixed via 5f0fdd1632bab19a2e9b3e577674ce6c979006ac